src/screens/Ticketing/index.tsx

```diff
@@ -6,5 +6,5 @@ import {useRemoteConfig} from '../../RemoteConfigContext';
 export default function Ticketing() {
   const {enable_ticketing} = useRemoteConfig();
 
-  return !enable_ticketing ? <Splash /> : <Tickets />;
+  return <Tickets />;
```

Comment:
I'd recommend enrolling to ticketing in simulator instead of commenting out this. Bit risky if it accidentally hits production.

Reply:
Enrolled now. Reverting this.